instrumentation/jmx-metrics/javaagent/src/main/resources/jmx/rules/camel.yaml

```diff
@@ -0,0 +1,262 @@
+---
+rules:
+  - bean: org.apache.camel:context=*,type=context,name=*
+    prefix: camel.context.
+    metricAttribute:
+      context: param(context)
+      camelVersion: beanattr(CamelVersion)
+    mapping:
+      ExchangesCompleted:
+        metric: exchange.completed
+        type: counter
+        unit: "{exchanges}"
+        desc: Indicates the total number of exchanges processed successfully since context start-up or the last reset operation.
+      ExchangesFailed:
+        metric: exchange.failed
+        type: counter
+        unit: "{exchanges}"
+        desc: Indicates the total number of exchanges that failed to process since context start-up or the last reset operation.
+      ExchangesInflight:
+        metric: exchange.inflight
+        type: updowncounter
+        unit: "{exchanges}"
+        desc: Indicates the number of exchanges currently transiting the context.
+      ExchangesTotal:
+        metric: exchange.total
+        type: counter
+        unit: "{exchanges}"
+        desc: Indicates the total number of exchanges, passed or failed, processed context start-up or the last reset operation.
+      FailuresHandled:
+        metric: failure.handled
+        unit: "{exchanges}"
+        type: counter
+        desc: Indicates the number of exchanges failed and handled by an ExceptionHandler in the context.
+      ExternalRedeliveries:
+        metric: external.redelivery.count
+        type: counter
+        unit: "{redeliveries}"
+        desc: The total number of all external redeliveries since context start-up or the last reset operation.
+      MaxProcessingTime:
+        metric: max.processing.time
+        unit: ms
+        type: gauge
+        desc: Indicates the longest time, in milliseconds, to process an exchange since context start-up or the last reset operation.
+      MeanProcessingTime:
+        metric: mean.processing.time
+        type: gauge
+        unit: ms
+        desc: Indicates the mean processing time, in milliseconds, for all exchanges processed since context start-up or the last reset operation.
+      MinProcessingTime:
+        metric: min.processing.time
+        unit: ms
+        type: gauge
+        desc: Indicates the shortest time, in milliseconds, to process an exchange since context start-up or the last reset operation.
+      LastProcessingTime:
+        metric: last.processing.time
+        unit: ms
+        type: gauge
+        desc: Indicates the time, in milliseconds, it took to process the last exchange.
+      DeltaProcessingTime:
+        metric: delta.processing.time
+        type: gauge
+        unit: ms
+        desc: Indicates the difference, in milliseconds, of the Processing Time of the last two exchanges transited the context.
+      TotalProcessingTime:
+        metric: total.processing.time
+        type: counter
+        unit: ms
+        desc: Indicates the total processing time, in milliseconds, to process all exchanges since context start-up or the last reset operation.
```

Comment:
I'd suggest putting these under a common namespace, e.g.

* `processing.max_time`
* `processing.mean_time`
* `processing.min_time`
* `processing.last_time`
* `processing.delta_time` (btw, is this even useful? i've not seen something like this before)
* `processing.time` (see `.time` at https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metrics.md#instrument-naming)

Reply:
yeah, looks more clean and neat in this way.